locator/tablets.cc

```diff
@@ -728,6 +728,16 @@ load_stats& load_stats::operator+=(const load_stats& s) {
     for (auto& [host, cap] : s.capacity) {
         capacity[host] = cap;
     }
+    if (s.critical_disk_utilization) {
+        if (!critical_disk_utilization) {
+            critical_disk_utilization = s.critical_disk_utilization;
```

Comment:
If the member wasn't optional than we'd need only the `else` loop below, simplifying this block.

Reply:
Agree.